src/main/java/org/casbin/casibase/entity/Record.java

```diff
@@ -0,0 +1,51 @@
+package org.casbin.casibase.entity;
+
+public class Record {
+
+    public int id;
```

Comment:
CI should be run, you can refer to casdoor-java-sdk or casvisor-java-sdk, copy their code (don't forget to modify project names)